docs/components/dialogs.md

```diff
@@ -0,0 +1,471 @@
+<!--docs:
+title: "Dialogs"
+layout: detail
+section: components
+excerpt: "Dialogs are modal windows that require interaction."
+iconId: 
+path: /catalog/dialog/
+-->
+
+
+# Dialogs
+
+Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
+
+There are four types of dialogs:
+
+1. [Alert](#alert-dialog)
+1. [Simple](#simple-dialog)
+1. [Confirmation](#confirmation-dialog)
+1. [Full-screen](#full-screen-dialog)
+
+<br>
+
+1. <img src="assets/dialogs/alert-dialog.png" alt="Alert dialog: discard" width=50%>
+1. <img src="assets/dialogs/simple-dialog.png" alt="Simple dialog: selection" width=50%>
+1. <img src="assets/dialogs/confirmation-dialog.png" alt="Confirmation dialog: selection confirmation" width=50%>
+1. <img src="assets/dialogs/full-screen-dialog.png" alt="Full-screen dialog: event" width=50%>
```

Comment:
Unless the migration tools have been updated, material.io can't handle <img> tags.

Use the composite image: https://drive.google.com/file/d/1Z0tfQAjumASNe2a0Todq2_l-MYLqikOa/view?usp=sharing

Reply:
done